packages/cf-forms/src/molecules/form-fields.less

```diff
@@ -174,15 +175,17 @@
             &:focus:checked + .a-label:before,
             &.focus:checked + .a-label:before {
                 border-color: @input-border__focused;
-                box-shadow: 0 0 0 1px @input-border__focused,
-                            inset 0 0 0 2px @form-field-inset;
+                box-shadow:
+                    0 0 0 1px @input-border__focused,
+                    inset 0 0 0 2px @form-field-inset;
```

Comment:
Instead of changing our customary format for properties like this, what do you think about adding `'declaration-colon-newline-after': null` to the rules list and modifying the `indentation` rule to:

```js
    'indentation': [
      4,
      {
        ignore: 'value'
      }
    ],
```

Reply:
👍 Sounds fine to me. Done!

Reply:
I've removed `font-family-no-missing-generic-family-keyword` also, since it's not applicable now that the webfont icons are gone.